prow/config.yaml

```diff
@@ -1123,7 +1123,7 @@ periodics:
       secret:
         secretName: auth-e2e-rbac-kubeconfig
```

Comment:
Actually, I should mention it in your latest PR. If we are not supporting 0.2, we can delete these clusters for old auth/pilot/mixer repo and of course, we don't need to cleanup them.

Reply:
done